src/apps/lwaftr/lwaftr.lua

```diff
@@ -19,9 +19,20 @@ local ffi = require("ffi")
 local band, bnot, rshift = bit.band, bit.bnot, bit.rshift
 local bitfield = lib.bitfield
 local C = ffi.C
+local cast = ffi.cast
+local receive, transmit = link.receive, link.transmit
+local empty, full = link.empty, link.full
 
 local debug = false
 
+local function compute_binding_table_by_ipv4(binding_table)
+   local ret = {}
+   for i,bind in ipairs(binding_table) do
```

Comment:
Nit: variable 'i' is not used in the loop, usually variable '_' is used as a placeholder for ignored variables.